organization/_posts/2022/2022-09-01-generators.md

```diff
@@ -0,0 +1,52 @@
+* Efe Yazgan: On slide 24, you mentioned the matrix element method. So, are there studies for ML that can potentially replace that. 
+    * AB: Hopefully soon. What we are working on conceptually is that you have a detector level event and you are sampling it from a gaussian and you get the corresponding probability distribution there. This is something you can do to unfold detector level events to invert to parton level. For MEM, given an event at parton level you look at the probabilities at the ME level where you have full control but you need a transfer function that gets you from parton to detector level (in the forward direction). You can train a network to learn this but since you start from events at the detector level  first you have to sample over every potential parton level event and then calculate the probability. This is numerically exteremely difficult that is why people start with a delta function which is not so correct approximation even for leptons but for jets it is much worse. People also use gaussians which is a better approximation but with NNs one has much more flexibility and more power to capture more compelx features. 
+    * EY: So, at this point, using MEM in a straightforward way is still faster than using ML?
+    * AB: Yes but many people have not worked on this so far. One has to give more time to include it and come out with standard packages.   
+* SJ: Just a follow up point on this. There is madminer and omnifold that kind of tries to do this. 
+    * AB: They are very different. Omnifold learns, with the classifier, the weights  between parton level, and detector level using full distributions just like the reweighting. And it applies them to parton level events to get the distribution. So, there is no unfolding of individual events. It massages the MC such that it looks like the data and then finds out what would my generation look like. Madminer uses again classifiers to learn these ratios but it does it in a different way. It directly uses the MEs as well to learn the ratios. 
+* MD: Follow up. I do not understand what you mean by direct event to event correlation. Unfolding only works for samples not on individual events with particles. How you do that on an event-by-event basis? 
+    * AB: slide 25; you have the events at the detector level and at the particle level. If I go through the usual chain of simulation from particle level to reco level standard MC run may times, for individual events on the RHS, I will get a distribution on the LHS. So, I can have P(SIM|GEN) or P(detector|particle) distribution. Using Bayes, theorem, I can express P(particle|detector) via P(detector|particle)xprior(particle)xprior(detector). Then, assume prior(particle), I have the other. The limitation is numerical for learning these high dimensional distributions - this is where we use the generative model. Second limitation is bias from the prior(particle) and one can use iterative approaches (which is work in progress).  
```

Comment:
This formatting breaks as the | is interprted as a sort of table separator. They need to be escaped, like:

```
P(SIM\|GEN)
```